virtcontainers/pkg/types/pcipath_test.go

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2020 David Gibson, Red Hat Inc.
```

Comment:
@dgibson - I think @devimc was referring to this one :smile:

Reply:
@dgibson yes, both headers are different...which one is the correct?

Reply:
Bother.  I forgot to update that one.  Done now.